src/azure/Templates/PageModelTemplate.cshtml

```diff
@@ -56,7 +56,7 @@ module @(Settings.Namespace)
       def get_all_items
         items = @@@Model.ItemName
         page = self
-        while page.@(Model.NextLinkName) != nil do
+        while page.@(Model.NextLinkName) != nil && !page.@(Model.NextLinkName).strip.empty? do
```

Comment:
why using "strip" here? are spaces the same as empty string, did you find cases where a string containing spaces got returned? I would just think that if we're accounting for the "empty" case, what comes back should really be empty.

Reply:
In an ideal world, I agree. But what if a service sends spaces? Java's NullNotEmpty check handles spaces too. So, I want to be in sync with them.